paasta_tools/cli/cli.py

```diff
@@ -115,13 +147,22 @@ def main(argv=None):
     Ensure we kill any child pids before we quit
     """
     logging.basicConfig()
+
+    # if we are an external command, we need to exec out early, and not
+    # let the subparser see all the extra args that it may not know about
```

Comment:
not sure i understand this comment. does it imply execing out early should be the last thing that happens in this function? because there is no `return` and parser will parse those args just few lines below.

Reply:
I've reworded.